src/webui/src/components/Modal/Compare.tsx

```diff
@@ -52,14 +52,14 @@ class Compare extends React.Component<CompareProps, {}> {
             tooltip: {
                 trigger: 'item',
                 enterable: true,
-                position: function (point: Array<number>, data: TooltipForIntermediate) {
+                position: function (point: Array<number>, data: TooltipForIntermediate): Array<number> {
```

Comment:
`Array<number>` is more oftenly written as `number[]`.

Reply:
Thanks